Model/Mesh_Statistics/MeshStatistics.cpp

```diff
@@ -0,0 +1,64 @@
+#include "MeshStatistics.h"
+#include <math.h>
+
+#include "../Common/CommonMeshRepresentation.h"
+
+namespace MeshIS {
+    namespace Model {
+        using Common::Element_Q4;
+        using Common::Element_T3;
+        using Common::Element_T4;
+        using Common::Element_P6;
+
+        void MS::processT3Elements() {
+            double dist1, dist2, dist3, halfPerimeter, inscribedCircleRadius, exscribedCircleRadius, ratio;
+            double sum = 0;
+            int i;
+            int *tempIDs = new int[3];
```

Comment:
Naked new warning!
It this case array size is constant, so just use:
int tempIDs[3];